documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -4,16 +4,14 @@ title: Cloudflare Pages
 
 To deploy to [Cloudflare Pages](https://developers.cloudflare.com/pages/), use [`adapter-cloudflare`](https://github.com/sveltejs/kit/tree/master/packages/adapter-cloudflare).
 
-This adapter will be installed by default when you use [`adapter-auto`](adapter-auto), but adding it to your project is recommended so that `event.platform` is automatically typed.
+This adapter will be installed by default when you use [`adapter-auto`](adapter-auto), but adding it to your project is recommended if you plan on staying with cloudflare pages.
```

Comment:
```suggestion
This adapter will be installed by default when you use [`adapter-auto`](adapter-auto). If you plan on staying with Cloudflare Pages you can switch from [`adapter-auto`](adapter-auto) to using this adapter directly so that type declarations will be automatically applied and you can set Cloudflare-specific options.
```

Reply:
Are the type declarations automatically applied? I tested that and it didn't seem to work :thinking:

Reply:
The code is definitely there, not sure why I can't get it to work locally � - but that's better in another PR